Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionDIFFTest.py

```diff
@@ -0,0 +1,59 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+from mantid.simpleapi import mtd
+from mantid import config
+from testhelpers import run_algorithm
+from mantid.api import MatrixWorkspace
+
+
+class IndirectILLReductionDIFF(unittest.TestCase):
+    # cache the def instrument and data search dirs
+    facility = config['default.facility']
+    instrument = config['default.instrument']
+    data_dir = config['datasearch.directories']
+
+    # Doppler
+    doppler_run = '276047'
+
+    def setUp(self):
+        # set instrument and append datasearch directory
+        config['default.facility'] = 'ILL'
+        config['default.instrument'] = 'IN16B'
+        config.appendDataSearchSubDir('ILL/IN16B/')
+
+        self.tolerance = 1e-2
```

Comment:
I think this is too loose. Set 1E-3 and make it relative (see sans tests for the flag).

Reply:
Actually remove that. This is a unit test, so there is no class wide tolerance.